setup.py

```diff
@@ -63,7 +63,7 @@ def apply_patches(self, base_dir):
 
     packages=['motioneye'],
 
-    install_requires=['tornado>=3.1,<6', 'jinja2', 'pillow', 'pycurl'],
+    install_requires=['tornado>=3.1,<6', 'jinja2', 'pillow', 'pycurl', 'minio'],
```

Comment:
Users shouldn't be forced to install `minio` in order to use motionEye. Having it as an option is fine, but let's not add it as a dependency.